hazelcast-jet-core/src/main/java/com/hazelcast/jet/Jet.java

```diff
@@ -38,7 +39,7 @@
  */
 public final class Jet {
 
-    private static final String INTERNAL_JET_MAPS = "__jet.*";
+    static final String INTERNAL_JET_MAPS = "__jet.*";
```

Comment:
could we make the Jet internal prefix public as well? There is another issue around this (#583)